tests/common/helpers/enable-dynamic-buffer.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```

Comment:
Can you please check if each part can be a function and one helper to call each in the right order? i think the code will be more readable in that way.

Reply:
Yes, we can. Will fix it.